app/controllers/api/v1/admin/trends/tags_controller.rb

```diff
@@ -1,11 +1,15 @@
 # frozen_string_literal: true
 
 class Api::V1::Admin::Trends::TagsController < Api::BaseController
+  include Authorization
+
   before_action -> { authorize_if_got_token! :'admin:read' }
-  before_action :require_staff!
   before_action :set_tags
 
+  after_action :verify_authorized
+
   def index
+    authorize :dashboard, :index?
```

Comment:
This API is used to display trends on the dashboard. I've decided to address this by making the API return public results when the user doesn't have the `manage_taxonomies` permission instead of throwing a HTTP 403.